imbi/endpoints/operations_logs.py

```diff
@@ -42,6 +58,88 @@ class CollectionRequestHandler(_RequestHandlerMixin,
                      %(version)s)
           RETURNING id""")
 
+    async def get(self, *args, **kwargs):
```

Comment:
Can we remove `*args`? It's not being used and at first I figured that our style guide required the use of `*args` preceding `**kwargs` but that does not seem to be the case.

Reply:
I don't have strong opinions on this. The other endpoints only use `self`, or have all three parameters and `*args` (and sometimes also `**kwargs`) are unused.

Reply:
I don't have a strong opinion either, it probably makes sense to stick with the established pattern.